courses/tests/test_dredd.py

```diff
@@ -0,0 +1,36 @@
+"""
+Test driver for Dredd
+"""
+import os
+import subprocess
+import shlex
+
+from django.conf import settings
+from django.test import LiveServerTestCase
+
+
+class DreddTestCase(LiveServerTestCase):
+    "Tests Dredd related tests."
+    def test_dredd(self):
+        """
+        Run test via Dredd.
+
+        We need to muck with the database url, among other things, to setup the
+        environment for testing properly, since we need to inform the dredd
+        script about our db config.
+        """
+        args = shlex.split('dredd {}/apiary.apib {} -f apiary_hooks.py --language python'.format(
+            settings.BASE_DIR, self.live_server_url))
+        old_db_url = os.getenv('DATABASE_URL')
+
+        # replace db name.
+        new_url = old_db_url[:-len('postgres')] + 'test_postgres'
```

Comment:
Can this be a string replace instead? Changes to the database connection string may break this